### ISSUE TITLE
Copy Paste not working

### ISSUE THREAD

**Issue by spv-pro** [warpdotdev/Warp#4604]
### Describe the bug

CTRL+SHIFT+C and CTRL+SHIFT+V is not working for copy and paste in WARP Terminal

### To reproduce

Go to terminal copy any cmd or paste the cmd

### Operating system

Linux

### Operating system and version

22.04

### Current Warp version

v0.2024.03.12.08.02.stable_02

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by Haz3lnut** [comment]
I found that simply double clicking a text item works to copy, but neither Ctrl-V nor Shift-INS work to paste. This is broken IMO. I am a new user, so maybe it's by design.

**Comment by acarl005** [comment]
Hi @Haz3lnut. Paste by default is ctrl-shift-V (to avoid a keybinding conflict with the SYN control char). We'll look into adding shift-ins though.

@spv-pro That's not good! Can you give us the following?
1. What is your keyboard layout and input method? We've had some problems with recognizing keybindings with some languages.
2. Close Warp, then from some other terminal which isn't Warp, can you run the  `warp-terminal` executable, try copy-pasting in the new Warp window, and sending us all the logs which appear?

**Comment by acarl005** [comment]
@spv-pro That's not good! Can you give us the following?
1. What is your keyboard layout and input method? We've had some problems with recognizing keybindings with some languages.
2. Close Warp, then from some other terminal which isn't Warp, can you run the  `warp-terminal` executable, try copy-pasting in the new Warp window, and sending us all the logs which appear?

**Comment by Haz3lnut** [comment]
Shift-INS is the expected paste key binding as it generally works in both GUI and terminal. I was not aware of ctrl-shift-V, but I now see it actually works in Gnome Terminal. I'm 20 years into Linux and never knew that; always used Shift-INS ¯\_(ツ)_/¯

Thanks!

**Comment by Haz3lnut** [comment]
I discovered we can set our own keybindings, so missing Shift-INS is not a problem for me anymore.

**Comment by spv-pro** [comment]
Actually, this problem is occurs sometimes while using Warp. When I close and reopen the Warp it's working fine

**Comment by kustodian** [comment]
Recommended would be to have both SHIFT+INS and CTRL+SHIFT+V, because the first one is the default terminal paste, while the second one is the new GUI paste which is cool if your keyboard has an INS key in very strange position :D

**Comment by AndreaDiazCorreia** [comment]
I have this issue on linux ubuntu 24

**Comment by saiema** [comment]
When copying from windows to warp in wsl2, ctrl + shift + v does not work for pasting, even after installing wslu.

**Comment by acarl005** [comment]
Hi all. Can you please share you Warp log file? It should be located at `${XDG_STATE_HOME:-$HOME/.local/state}/warp-terminal/warp.log`

**Comment by ProoGess** [comment]
> Hi all. Can you please share you Warp log file? It should be located at `${XDG_STATE_HOME:-$HOME/.local/state}/warp-terminal/warp.log`

Linux mxwork 6.0.0-6mx-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.0.12-1~mx21+1 (2022-12-15) x86_64 GNU/Linux

/.local/state/warp-terminal/warp.log: command not found

CTLR+V & SHIFT+INSSERT = not work

**Comment by pkumaschow** [comment]
Yeah this happening on Fedora 41 as well. Trying to cut and paste from Firefox. 

OS: Fedora Linux 41 (Workstation Edition) x86_64
DE: GNOME 47.4
WM: Mutter (Wayland)

warp-terminal was updated a couple of hours ago.... can't seem to find the version I am running though. 

Hmmm: warp-terminal: /lib64/libcurl.so.4: no version information available (required by warp-terminal

**Comment by zloeber** [comment]
Super frustrating. Copy and paste are probably the most basic requirements for this to be a functional application.  I do not even get a paste option in the right-click context window. This specifically occurs for prompted input for other applications (in my case running `goose configure` ). But just run `nano` and try to paste anything at all via keyboard or right-click to replicate. 

OS: Linux Mint 22.1 (Xia)

**Comment by Skaz7** [comment]
I had the same problem and i managed to solve it on Ubuntu 24.10
In WarpTerminal `Settings->Features->System-> Honor linux selection clipboard settings` I unchecked it and checked it again, then pasting started working properly with CTRL+SHIFT+V

**Comment by adsick** [comment]
I encountered the same issue on Fedora 42 (Wayland). Tried previous fix, didn't work right away, but started working after a Warp restart. Interestingly it works even without that option unchecked. Some unreliable s*ht.

**Comment by saintdron** [comment]
I have the same problem. No keyboard shortcuts paste text. Even in the context menu, sometimes there is no corresponding item. And even if there is, it still doesn't paste. In any other application, the same text is inserted. Only restarting Warp helps.

**Comment by Zarubaxa** [comment]
> I had the same problem and i managed to solve it on Ubuntu 24.10 In WarpTerminal `Settings->Features->System-> Honor linux selection clipboard settings` I unchecked it and checked it again, then pasting started working properly with CTRL+SHIFT+V

maaaaaan! your advice is working! thanks!)

**Comment by saintdron** [comment]
> Sometimes I select text in the console, press CTRL+C, and instead of copying, it enters ^C. Other times, it copies the selected text.

Same here. After that, I started copying almost exclusively through the context menu. Because I closed several processes that I didn't intend to.

**Comment by JacobNWolf** [comment]
FYI for Linux users, you can change this in Settings > Keyboard Shortcuts. Then Ctrl + V will work.

**Comment by Dazag** [comment]
I can copy, it doesn't paste into my jetbrains IDE....

**Comment by dextergiii** [comment]
Moving away from warp as I couldn't copy and paste an authorization URL on Windows 11.
